docs/_includes/common/cdn.html

```diff
@@ -1,5 +1,7 @@
 <div id="cdn">
     <h2>CDN</h2>
-    <p>A CSS file containing the full collection of modules is available via the following url:</p>
-    <p><a href="http://ir.ebaystatic.com/cr/v/c1/skin/v6.3.2/ds4/skin.min.css">https://ir.ebaystatic.com/cr/v/c1/skin/v6.3.2/ds4/skin.min.css</a></p>
+    <p>A CSS file containing the full collection of modules (except background icon data-urls) is available via the following url:</p>
+    <p><a href="http://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin.min.css">https://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin.min.css</a></p>
+    <p>Due to their large size, background icon data-urls are considered optional, and are available separately via the following url:</p>
+    <p><a href="http://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin-base64.min.css">https://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin-base64.min.css</a></p>
```

Comment:
Should these URLs be changed to `v7.0.0` in anticipation? We won't publish the separate base64 version until then.

Reply:
Yeah, that makes sense. I'll change them now.

Reply:
Done.

Reply:
👍